src/hooks/useFileUpload.tsx

```diff
@@ -238,8 +266,19 @@ export default function useFileUpload(
     if (!validateFileUpload()) return;
 
     setProgress(0);
+    let filesToUpload = files;
+
+    if (files.length > 1) {
```

Comment:
Hmm, right now it would combine any multi-uploads, let's not do that..instead let's add an option on FE (similar to "Upload From Device, Open Camera" etc) to trigger this.